firmware-src/sources/dhdebug.c

```diff
@@ -26,15 +26,16 @@ LOCAL char isDirect = 0;
 LOCAL void ICACHE_FLASH_ATTR recover_led(void *arg) {
 	ETS_INTR_LOCK();
 	PIN_FUNC_SELECT(PERIPHS_IO_MUX_U0TXD_U, FUNC_GPIO1);
-	gpio_output_set(0, 0x2, 0x2, 0);
+	gpio_output_set(0, 0b0110, 0b0110, 0);
```

Comment:
Different boards have LED connected to UART0TX OR UART1TX. Since both pins are used for output on booting, it's possible to set up low level on both of them while AP mode is activated.